script/utils/JsonRegistry.s.sol

```diff
@@ -15,15 +18,19 @@ contract JsonRegistry is Script {
         registeredContracts += 1;
     }
 
-    function startDeploymentOutput() public {
+    function startDeploymentOutput(bool isTests) public {
         deploymentOutput = '{\n  "contracts": {\n';
+
+        if (!isTests) {
+            console.log(
```

Comment:
Making it easier to find the latest file

Reply:
NIT. Maybe instead of passing `isTests` for all the files, we can just use an environmental variable? So only from tests we do:

```
vm.setEnv("IS_TESTS", 1);
```

That way we leave deployer interface untouched with testing stuff